Backpack/Rating/Classes/BPKRating.swift

```diff
@@ -0,0 +1,263 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import UIKit
+
+public class BPKRating: UIView {
+
+    // MARK: - Public properties
+    public var value: Float {
+        didSet { updateLookAndFeel() }
+    }
+    public var ratingScale: BPKRatingScale {
+        didSet { updateLookAndFeel() }
+    }
+    public var size: BPKRatingSize {
+        didSet { updateLookAndFeel() }
+    }
+
+    public var showScale: Bool {
+        didSet { updateLookAndFeel() }
+    }
+    public var titleView: UIView {
+        didSet { updateLookAndFeel() }
+    }
+
+    public var title: String {
+        get {
+            titleLabel.text ?? ""
+        }
+        set {
+            titleLabel.text = newValue
+            titleView = titleLabel
+            updateLookAndFeel()
+        }
+    }
+    public var subtitle: String? {
+        get {
+            subtitleLabel.text ?? ""
+        }
+        set {
+            subtitleLabel.text = newValue
+            updateLookAndFeel()
+        }
+    }
+
+    // MARK: - Subviews
+    private let horizontalStackView: UIStackView = {
+        let horizontalStack = UIStackView()
+        horizontalStack.axis = .horizontal
+        horizontalStack.spacing = BPKSpacingMd
+        horizontalStack.alignment = .lastBaseline
+        return horizontalStack
+    }()
+    private let ratingValueAndScaleStackView: UIStackView = {
+        let horizontalStack = UIStackView()
+        horizontalStack.axis = .horizontal
+        horizontalStack.alignment = .lastBaseline
+        return horizontalStack
+    }()
+    private let titleSubtitleStackView: UIStackView = {
+        let verticalStack = UIStackView()
+        verticalStack.axis = .vertical
+        verticalStack.alignment = .leading
+        verticalStack.distribution = .fill
+        return verticalStack
+    }()
+
+    private let ratingValueLabel: BPKLabel = {
+        let label = BPKLabel(fontStyle: .textLabel1)
+        label.textColor = BPKColor.textPrimaryColor
+        return label
+    }()
```

Comment:
```suggestion
    private lazy var ratingValueLabel: BPKLabel = BPKLabel(fontStyle: .textLabel1)
```
A BPKLabel has `.textPrimary` as a default. Saves you some lines of code :D

Reply:
The related commit is d0ee6ed296b3b29f6054ed34cf8afd7117bfbfcc